### ISSUE TITLE
New warnings in 0.11.0 (ClojureScript)

### ISSUE THREAD

**Issue by stoyle** [nathanmarz/specter#110]
I guess these are all innocuous warnings:

```
WARNING: Use of undeclared Var cljs.js/*eval-fn* at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: cljs.js/*eval-fn* not declared ^:dynamic at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/*eval-fn* at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/*eval-fn* at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/js-eval at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/*eval-fn* at line 52 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/eval at line 53 tests-out/com/rpl/specter/impl.cljs
WARNING: Use of undeclared Var cljs.js/empty-state at line 53 tests-out/com/rpl/specter/impl.cljs
```

However we do fail our builds when the ClojureScript compiler emits warnings, since they're almost always errors. Would be nice to have these fixed.

Keep up the good work! :)
Cheers,
Alf

**Comment by nathanmarz** [comment]
Already fixed for next version, see #97. Will leave this issue open until that release.

**Comment by stoyle** [comment]
Oh, sorry. Searched for "WARNING", but I guess I only did it in open issues. Good idea to keep this open, then.
